qa-data/invalid-data-xlsx/datapackage.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "invalid-data-xlsx",
+    "resources": [
+        {
+            "name": "data",
+            "path": "data.xlsx",
```

Comment:
@anuveyatsu It is valid, but we do not support `.xlsx`. Do we?  that is why `invalid-data`